src/benchmark.rs

```diff
@@ -391,7 +391,10 @@ pub fn run_benchmark(
     // Compute statistical quantities
     let t_num = times_real.len();
     let t_mean = mean(&times_real);
-    let t_stddev = standard_deviation(&times_real, Some(t_mean));
+    let mut t_stddev = 0.0;
+    if *&times_real.len() != 1 {
```

Comment:
It would be great if you could fix the clippy warning (see below). There is a similar warning below.

Reply:
Fixed, thanks. No idea those were a thing, more Rust beginner mistakes...